### PR TITLE
✨ Add new supporter template

### PR DESCRIPTION
**Pull request by nwingt**
@williamchong @nnkken

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,21 @@
+MIT License
```
Comment (williamchong) [inline LICENSE:1]: We generally use GPLv3 in likecoin repos

File: .github/workflows/main.yml [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+on: [push]
+jobs:
+  build:
+    name: Build, lint, and test on Node ${{ matrix.node }} and ${{ matrix.os }}
+
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        node: ['10.x', '12.x', '14.x']
+        os: [ubuntu-latest, windows-latest, macOS-latest]
```
Comment (williamchong) [inline .github/workflows/main.yml:11]: We probably don't need to run this in matrix, just ubuntu with LTS node version would do
Most other repos are still at 10 but maybe we will upgrade later

File: README.md [hunk 3]
```diff
@@ -0,0 +1,103 @@
+# TSDX User Guide
```
Comment (williamchong) [inline README.md:1]: Maybe we should at least update the README's title

File: .github/workflows/main.yml [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+        uses: bahmutov/npm-install@v1
+
+      - name: Lint
+        run: yarn lint
+
+      - name: Test
+        run: yarn test --ci --coverage --maxWorkers=2
+
+      - name: Build
+        run: yarn build
```
Comment (williamchong) [inline .github/workflows/main.yml:32]: Seems this step is producing an error in your fork

File: src/index.ts [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+}) => {
+  const options = {
+    priceEmojiURL: getPriceEmojiURL(opts.amount),
+    ...opts,
+  }; 
+  // Compile MJML
+  const template = compileTemplate(getTemplateFile('new-supporter.mjml'), options);
+  // Compile HTML
+  const html = parseMJMLToHTML(template);
+  return compileTemplate(html, options);
```
Comment (williamchong) [inline src/index.ts:51]: After MJML is compiled to HTML, why do we need to compile it again?

**Review by williamchong — COMMENTED** [review]
How will we "use" this repo? Include this and calling `getNewSupporterTemplate` from other services?

**Comment by nwingt** [comment]
I think there is a bug new commits are missing for changing repo visibility after opening a PR, will open another PR

**Pull request closed without merging**